Kernel/StackWriter.h

```diff
@@ -0,0 +1,59 @@
+/*
+ * Copyright (c) 2022, Kim Nilsson <kim@wayoftao.net>
+ *
+ * SPDX-License-Identifier: BSD-2-Clause
+ */
+
+#pragma once
+
+#include <AK/StdLibExtraDetails.h>
+#include <AK/Types.h>
+
+namespace Kernel {
+
+class StackWriter {
```

Comment:
(Also nit you don't really need to address, this writer does a whole lot of reading)